gradle.properties

```diff
@@ -1,12 +1,12 @@
 # Done to increase the memory available to gradle.
 org.gradle.jvmargs=-Xmx1G
 
-minecraft_version=1.19.3
-yarn_mappings=1.19.3+build.2
-loader_version=0.14.11
+minecraft_version=1.19.4-rc2
+yarn_mappings=1.19.4-rc2+build.1
+loader_version=0.14.17
 
 #Dependencies
-fabric_version=0.68.1+1.19.3
+fabric_version=0.75.3+1.19.4
 
 # Mod Properties
 mod_version=0.3.4
```

Comment:
Bump version? 🙂

Reply:
Should it be bumped to 0.3.5 or 0.4.0? When updating from Minecraft 1.19.1 to 1.19.3, the version stayed the same.

Reply:
Ahh, probably should have bumped then too - but I think 0.3.5 makes sense to me 🙂

Reply:
Fixed in 5aeb45e2b5032ad3dfe59cd52e3ccc011dd95d18